tensorflow/compiler/jit/xla_gpu_device.cc

```diff
@@ -51,14 +51,18 @@ Status XlaGpuDeviceFactory::CreateDevices(const SessionOptions& options,
   return Status::OK();
 }
 
+#if GOOGLE_CUDA
```

Comment:
@hawkinsp Thank you for your suggestion!

 I will surely follow this to update this PR.